mobify/platform-scaffold
# WEB-1499: ios Add to Homescreen fixes

This PR updates the title and icon used by iOS when adding to the homescreen. You'll need to do some set up in Charles otherwise the local image won't work in iOS. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1499
 **Linked PRs**: n/a

## Changes
- Added iOS app icon
- Set iOS homescreen app icon in loader.js
- Set iOS homescreen app icon title in loader.js

## Todos

## How to test-drive this PR
- in Charles enable SSL Proxying (Proxy menu -> SSL Proxy Settings)
![screen shot 2017-08-14 at 11 58 25 am](https://user-images.githubusercontent.com/1301857/29287071-0ee716d2-80e9-11e7-86d7-ed9151efff06.png)
- Also in Charles, install the root certificate on your iOS simulator (Help Menu -> SSL Proxying -> Install Root Certificate in iOS Simulators)
![root certificate](https://user-images.githubusercontent.com/1301857/29287234-9fa805aa-80e9-11e7-973b-56d24972200c.png)
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1) in the iOS simulator
- Click the share button on the bottom bar of your browser (middle button)
- Click Add to Home screen in the share menu
- In the prompt that opens you should see an image with the Merlins potions icon in it and the title should default to "Merlin's Potions"
